google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddAdCustomizer.java

```diff
@@ -141,7 +143,7 @@ private void runExample(GoogleAdsClient googleAdsClient, AddAdCustomizerParams p
           "Please pass exactly two ad group IDs in the adGroupId parameter.");
     }
 
-    String feedName = "Ad Customizer example feed " + System.currentTimeMillis();
+    String feedName = "Ad Customizer example feed " + getPrintableDatetime();
```

Comment:
@PierrickVoulet , could you trigger the functional tests against this PR to make sure that none of the names exceed product limits for an entity? With this change, we're going from around 13 characters (system millis) to 25.

Reply:
Good call, I few were failing due to size.  I added a shorter version of the printable data time (15 char-long) and used it for those that needed it.

Reply:
All functional tests pass, only the AddCampaignDraft fails but for unrelated issue.